packages/remark-mdxjs/test/__snapshots__/test.js.snap

```diff
@@ -1,51 +1,88 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`correctly transpiles 1`] = `
-"import Foo from './bar'
+exports[`Handles an import that is later exported 1`] = `
```

Comment:
general question, probably don't require changes.
In some places you opt for tests in others snapshots.
What is the thinking about which should be used where?

Reply:
Snapshots were originally there. I can see some use in them, but I personally generally prefer them inline and just as a normal test.